data/hai/hai reveal 4 middle.txt

```diff
@@ -1233,29 +1233,77 @@ mission "Hai Reveal [A11] Sound the Alarm"
+				goto "the elders vote"
+			label "turn back unfettered"
+			action
+				set "told hai to reject unfettered help"
+			`	A disgruntled Elder responds, "Change our decision? Our decision not to violate human sovereignty? Even if I lose my Eldership, I will not vote for violence."`
+			`	Teeneep says, "If you take no action, and refuse help from our brethren, they'll pass through our territory anyway. We can hold back Unfettered raids, but their entire armada? That is the unnecessary violence you bring through inaction."`
+			label "the elders vote"
+			`	Another Elder interjects, "You're exaggerating the impact, Teeneep. Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone."`
+			`	Many Elders seem appalled by that observation, but the moderator interrupts. "We will vote on two counts: Do we act? Do we accept Unfettered help?"`
+			choice
+				`	"What's going to happen, Teeneep?"`
+					goto teeneep
+				`	"Sayari, what happens next?"`
+			`	Sayari explains, "There's a multi-step voting process, which takes into account dissent between Elders. After each stage of voting, there's discussions between strategically chosen groups of Elders, and afterwards the dissent is usually lower. They iterate until they come to a consensus, unless it takes too long; then they'll use the majority vote."`
+				goto results
+			label teeneep
+			`	Teeneep responds, "They'll vote to do nothing. It's all this council knows how to do." Sayari sighs loudly at this but doesn't contradict Teeneep.`
+			label results
+			`	After a few more minutes of private discussions, the Elders vote "no" on both counts. Your words may not have swayed the vote, but perhaps they'll resonate in the Hai psyche long enough to have an effect.`
+			`	Teeneep is allowed to respond and says ominously, "You know what will happen next."`
+			`	An angry Elder replies, "You wouldn't dare!" Teeneep yells back, "It is already scheduled to happen unless I actively stop it!" She smiles, a little self indulgently, and withdraws from the room. Sayari follows, motioning for you to do so as well.`
+			`	Sayari leads you to a private meeting room.`
+			`	Teeneep explains her ominous words. "Our democracy allows a Petition of No Confidence against the Elders. If there are enough petitioners, it forces an election. Activists in the organization I founded have led a grassroots movement pushing a petition for weeks.`
+			`	"It was obvious what the Elder Council's response would be," and with a bit of sarcasm she adds, "and I knew I would only come back alive if the Unfettered agreed to help. So, my supporters arranged in advance for the petition to be promoted along with a widespread broadcast of these proceedings."`
+			`	Sayari assures you, "These petitions take time and rarely succeed. It is more of a warning to the Elders than anything else. Still, we should watch the situation.`
```

Comment:
One of the objectives of the original piece was to show the Hai as a politically engaged people, but a people that are undecided.
A people equipped with a mechanism for making direct and radical change, but a predilection for disagreeing on thing sufficiently that it rarely amounted to being more than an opinion poll for the Elder's to observe the results of.

This also has issues with being presented as a grassroots movement:

- Such movements tend to take more than a few weeks.
- The Hai population is not small, to push a movement like that quickly would require Teeneep's organisation to be *very* large. (Which is why originally she was leveraging existing infrastructure, with the aid of her organisation's influence.)
- You know the whole bit about needing to confront the Elders directly and conform to the process? That is *directly* at odds with the concept of making sensitive information publicly available in the background as part of some social campaign.
- Corollary: if that campaign were successful enough to already be at this stage, the Elders would have heard about it already and she'd have been summarily censured for breach of process and betrayal of trust.